frontend/src/LoginForm.js

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { Formik, Form } from "formik";
+import * as yup from "yup";
+
+import EmailInput from "./EmailInput";
+import PasswordInput from "./PasswordInput";
+import Button from "./Button";
+//const axios = require('axios')
+var axios = require("axios");
+var MockAdapter = require("axios-mock-adapter");
```

Comment:
Itt is használhatunk ES6-os importokat nyugodtan, gondolom az example miatt lett így.

Reply:
módosítva

Reply:
Nem látom még, pusholtad?

Reply:
Most már igen. :)